OOP_ComposerLessons/Lesson13/src/User.php

```diff
@@ -0,0 +1,58 @@
+<?php
+namespace Lesson13;
+/**
+ * [Description User]
+ */
+class User
+{
+	private string $surname; // фамилия
+	private string $name; // имя
```

Comment:
Useless comments. If you gave proper name to your variables, it's no need in comments